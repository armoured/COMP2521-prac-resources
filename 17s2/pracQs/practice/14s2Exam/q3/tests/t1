./q3 < tests/t1.in
