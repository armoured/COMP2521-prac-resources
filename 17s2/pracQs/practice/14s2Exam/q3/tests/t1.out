Graph:
# vertices = 2, # edges = 1

0 is connected to 1
1 is connected to 0

No well-connected vertices
